extra_data/tests/test_reader_mockdata.py

```diff
@@ -615,13 +615,24 @@ def test_open_file(mock_sa3_control_data):
 
 
 def test_permission():
+    #  If users is root then `PermissionError` won't be raised, set the euid to
+    #  standard user so the tests work correctly
+    euid_was_root = False
+    if hasattr(os, 'seteuid'):
+        if os.geteuid() == 0:
+            os.seteuid(1000)
```

Comment:
Somewhat to my surprise, this appears to work (at least on Linux) even with a UID that doesn't exist.